generators/app/templates/documentation/paths/index.js

```diff
@@ -0,0 +1,3 @@
+const user = require('./user');
+
+module.exports = user;
```

Comment:
If multiple paths are used we need to export like this `module.exports = { ..user, ...other }`?

Reply:
yes